config/config.go

```diff
@@ -278,6 +285,18 @@ func (s *SourceSubnet) Validate(destinationBlockchainIDs *set.Set[string]) error
 		s.supportedDestinations.Add(blockchainID)
 	}
 
+	// Validate the starting block height and set the bigint field
+	// If -1 is provided, set the bigint field to 0 to indicate that the relayer should start from genesis
+	// 0 is treated the same as the field being unset, since there's no way to differentiate between the two when unmarshalling
+	if s.StartBlockHeight < -1 {
+		return fmt.Errorf("invalid start block height in source subnet configuration. Provided height: %d", s.StartBlockHeight)
+	}
+	if s.StartBlockHeight == -1 {
+		s.StartBlockHeightBigInt = big.NewInt(0)
+	} else if s.StartBlockHeight > 0 {
+		s.StartBlockHeightBigInt = big.NewInt(s.StartBlockHeight)
+	}
```

Comment:
Can you leave a separate comment saying that if `StartBlockHeight` is 0, then `StartBlockHeightBigInt` will be nil?